src/dynamixel_hardware_interface.cpp

```diff
@@ -51,6 +52,22 @@ namespace dynamixel {
             throw e;
         }
 
+        // remove servos that are not in the _dynamixel_map (i.e. that are not used)
+        std::vector<dynamixel::DynamixelHardwareInterface::dynamixel_servo>::iterator servo_it;
+        for (servo_it = _dynamixel_servos.begin(); servo_it != _dynamixel_servos.end();) {
+            std::map<long long int, std::string>::iterator dynamixel_iterator = _dynamixel_map.find((*servo_it)->id());
+            if (dynamixel_iterator == _dynamixel_map.end()) // the actuator's name is not in the map
+                servo_it = _dynamixel_servos.erase(servo_it);
+            else
+                ++servo_it;
+        }
+
+        // Check that no actuator was declared by user but not found
+        if (_dynamixel_map.size() > _dynamixel_servos.size()) {
+            ROS_FATAL_STREAM("Some servos were declared in the parameters but could "
+                             "not be found");
+        }
```

Comment:
My bad, I had not disabled the controllers. Doing so doesn't make anything crash. I assume that it's what you do when you remove some actuators on the robot, isn't it ?